### PR TITLE
feat: Add is_rhel_ai into parser os_release

### PR DESCRIPTION
**Pull request by wushiqinlou**
rh-pre-commit.version: 2.3.1
rh-pre-commit.check-secrets: ENABLED

### All Pull Requests:

Check all that apply:

* [x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [x] No Sensitive Data in this change?
* [ ] Is this PR to correct an issue?
* [ ] Is this PR an enhancement?

### Complete Description of Additions/Changes:
Add `is_rhel_ai` function to check if the system is from RHEL AI image

### CODE REVIEW

File: insights/parsers/os_release.py [hunk 1]
```diff
@@ -67,3 +67,11 @@ def data(self):
...
         """
         return self
+
+    @property
+    def is_rhel_ai(self):
+        """
+        Returns:
+            bool: True when the system is from RHEL AI image.
+        """
+        return True if self.get("VARIANT", None) == "RHEL AI" and self.get("VARIANT_ID", None) == "rhel_ai" else False
```
Comment (xiangce) [inline insights/parsers/os_release.py:77]: nitpick, how about 
```python
return self.get("VARIANT") == "RHEL AI" and self.get("VARIANT_ID") == "rhel_ai"
```

Comment (wushiqinlou) [reply insights/parsers/os_release.py]: @xiangce - updated, please confirm

File: insights/parsers/os_release.py [hunk 2]
```diff
@@ -67,3 +67,11 @@ def data(self):
             Deprecated, it will be removed from 3.7.0
         """
         return self
+
+    @property
+    def is_rhel_ai(self):
```
Comment (JoySnow) [inline insights/parsers/os_release.py:72]: There are also a combiner `OSRelease` from `insights/combiners/os_release.py`. 
Shall we also propagate this `is_rhel_ai` into the combiner?  

@xiangce any suggestion on this?

Comment (xiangce) [reply insights/parsers/os_release.py:72]: As the `OSRelease` combiner is majorly for identify if the host an RHEL OS system.  
https://github.com/RedHatInsights/insights-core/blob/644d314cd574da23dd3d52d24f78bb8b4b84ddcd/insights/combiners/os_release.py#L4-L5
And the "is_rhel_ai" can completely being decided by the "Specs.os_release" alone.
It's not so necessary to add this interface to the Combiner, in my opinion.

But it's up to you, depends on how rule devs preference. I'd leave it to @JoySnow and @wushiqinlou

Comment (wushiqinlou) [reply insights/parsers/os_release.py:72]: @JoySnow - Let's keep monitoring. If it is necessary to add the func into combiner during the development in the future, I will add it.

**Review by xiangce — APPROVED** [review]

**Review by JoySnow — APPROVED** [review]

**Pull request merged**